### PR TITLE
Susan

### PR DESCRIPTION
**Pull request by slovelan**

### CODE REVIEW

File: carta/cpp/CartaLib/ICoordinateFormatter.h [hunk 1]
```diff
@@ -56,6 +56,9 @@ class CoordinateFormatterInterface {
     /// enable printing of an axis
     virtual Me & enableAxis(int ind) = 0;
 
+    /// Set the axes the user will see (3 axis: x,y axes plus a hidden axis).
+    virtual void setDisplayAxes( std::vector<Carta::Lib::AxisInfo::KnownType> displayAxes ) = 0;
+
```
Comment (pfederl) [inline carta/cpp/CartaLib/ICoordinateFormatter.h]: Does this have to be part of the interface class? I think the functionality could be entirely implemented already using just the public methods of this class.

File: carta/cpp/CartaLib/IImage.h [hunk 2]
```diff
@@ -319,6 +321,9 @@ class ImageInterface
     virtual const VI &
     dims() const = 0;
 
+    /// return the index of the specified axis in the image.
+    virtual int getAxisIndex( Carta::Lib::AxisInfo::KnownType axis ) const = 0;
+
```
Comment (pfederl) [inline carta/cpp/CartaLib/IImage.h]: Does this have to be part of the interface class? I think the functionality could be entirely implemented already using just the public methods of this class.

File: carta/cpp/CartaLib/IWcsGridRenderService.h [hunk 3]
```diff
@@ -67,6 +67,19 @@ class IWcsGridRenderService : public QObject
...
+
+    // set on which edge (top,bottom,left,right) the given axis should
+    // be drawn; setting the edge to "" will remove labelling for the axis.
+    virtual void
+    setAxisLabelLocation( int axisIndex, const QString& edge ) = 0;
+
+    // set the format to use when labelling an axis.
+    virtual void
+    setAxisLabelFormat( int axisIndex, const QString& formatStr ) = 0;
+
```
Comment (pfederl) [inline carta/cpp/CartaLib/IWcsGridRenderService.h]: I think we should probably describe what is the format of the formatStr

File: carta/cpp/CartaLib/ICoordinateFormatter.h (hunk 1 above)
Comment (pfederl) [inline carta/cpp/CartaLib/ICoordinateFormatter.h]: Can we remove this API from the interface?

Comment (low-sky) [inline carta/cpp/CartaLib/ICoordinateFormatter.h]: @slovelan -- Is this still an issue?  Is it a major fix?  I think this PR is good to go if this is cleaned up.

**Comment by low-sky** [comment]
@slovelan notes that the unused methods have been cleaned up in a future PR that will include new progress with axis work.  Given it doesn't affect functionality, we can go ahead and do merge.  I'm happy to proceed like that.  

@astrilet  -- I think we are good to merge to develop, run builds and tag to master as v0.4

**Pull request merged**